packages/server/lib/routes.ts

```diff
@@ -192,33 +197,43 @@ publicAPI.route('/auth/bill/:providerConfigKey').post(connectSessionOrPublicAuth
 publicAPI.route('/auth/signature/:providerConfigKey').post(connectSessionOrPublicAuth, postPublicSignatureAuthorization);
 publicAPI.route('/auth/unauthenticated/:providerConfigKey').post(connectSessionOrPublicAuth, postPublicUnauthenticated);
 
+publicAPI.use('/unauth', jsonContentTypeMiddleware);
 // @deprecated
 publicAPI.route('/unauth/:providerConfigKey').post(connectSessionOrPublicAuth, postPublicUnauthenticated);
 
+publicAPI.use('/webhook', jsonContentTypeMiddleware);
```

Comment:
unfortunately it looks like not all webhooks sends json :( https://github.com/NangoHQ/nango/blob/b1c0f9a2f44d594dd29153f0c2e8ae72c99b550e/packages/server/lib/webhook/slack-webhook-routing.ts#L8

Reply:
ah yes thanks for catching that